### DISCUSSION TITLE
Dont commit public folder bcz it gets regenerated so causes big diff in git but specify favicon.ico in Next.js?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [vercel/next.js#35037] [Help] [answered]
I am making a blog that regenerates the images with different names like `image-random-string-like-312asdaczx` so causes a big diff in my commit.

I had already put `public/` folder in my `.gitignore` but I want to use `favicon.ico` so I had to remove it.

Is there a way to use `favicon.ico` without having to commit `public/` folder?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This is not necessarily Next.js related, but I think you could tweak your `.gitignore` file to:

```
/public/*
!/public/favicon.ico
```

**Reply by deadcoder0904** [reply]
thanks, i hadn't used this in years :)
